apps/api/src/app/environments-v1/usecases/output-renderers/hydrate-email-schema.usecase.ts

```diff
@@ -234,3 +233,5 @@ export const TipTapSchema = z.object({
   text: z.string().optional(),
   attrs: z.record(z.unknown()).optional(),
 });
+
+const buildLiquidJSDefault = (mailyFallback: string) => (mailyFallback ? ` | default: '${mailyFallback}'` : '');
```

Comment:
Consider combining this with `const finalValue = `{{ ${variableName} ${buildLiquidJSDefault(fallback)} }}`;` under a `buildLiquidJSVariable`.